c2cgeoportal/tests/functional/test_mapserverproxy_capabilities.py

```diff
@@ -162,10 +154,6 @@ def _wms_get_capabilities_config(ogcserver, service="wms", username=None):
                 "default_ogc_server": ogcserver,
             },
         })
-        request.params = {"map": os.path.join(
-            os.path.dirname(os.path.abspath(__file__)),
-            "c2cgeoportal_test.map"
```

Comment:
Can't you remove the "username" part below as you have done that in other part of the code ?